docker-app/qfieldcloud/core/admin.py

```diff
@@ -568,6 +568,27 @@ class ProjectFilesWidget(widgets.Input):
     template_name = "admin/project_files_widget.html"
 
 
+class OwnerTypeFilter(admin.SimpleListFilter):
+    # "owner__type" for the filter would suffice,
+    # But the filter title would then be "By type" in the UI,
+    # which is semantically unclear, hence the SimpleListFilter subclass
+    title = "owner type"
+    parameter_name = "owner_type"
+
+    def lookups(self, request, model_admin):
+        return [
+            (User.Type.PERSON, "Person"),
+            (User.Type.ORGANIZATION, "Organization"),
+            (User.Type.TEAM, "Team"),
+        ]
+
+    def queryset(self, request, queryset):
+        if self.value():
+            return queryset.filter(owner__type=self.value())
+        else:
+            return queryset
```

Comment:
Here I would prefer the same style as in other SimpleListFilters
 
```
       ...
        value = self.value()

        if value is None:
            return queryset
            
         ... rest of logic
```

Reply:
done